dotnet/src/Connectors/Connectors.OpenAI/AzureSdk/OpenAIFunction.cs

```diff
@@ -101,7 +101,7 @@ public sealed class OpenAIFunction
     }
 
     /// <summary>Gets the separator used between the plugin name and the function name, if a plugin name is present.</summary>
-    public static string NameSeparator => "_";
+    public static string NameSeparator { get; set; } = "-";
```

Comment:
Can you provide some background on this change?

I think we would be useful to add a `<remarks>` section to the API documentation explaining why we are using `-` in preference to `_`.

Reply:
There had been some discussions in December about aligning the two planners to use the same separator, and agreement that dash would be the better option (we've seen some issues from people trying to use underscores/snake case in their function names). However, the stepwise planner was never updated after those discussions. I agree, adding to the API documentation would be a good idea - will update.